examples/guestbook/frontend-controller.yaml

```diff
@@ -1,12 +1,12 @@
-apiVersion: v1
```

Comment:
That sounds reasonable.  This may be less obvious to our users, but I also think they will just accept the name without giving it a lot of thought.